data/rulei/rulei.txt

```diff
@@ -11,18 +11,20 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
-mission "Relic of the Past"
```

Comment:
Is the point of this simply to force the player to get the mission again to replace the ship?

Reply:
I saw that, and figured that was part of the reason. Thanks for confirming.